src/AppInstallerCLICore/Workflows/ManifestComparator.cpp

```diff
@@ -11,29 +11,28 @@ namespace AppInstaller::CLI::Workflow
     bool InstallerComparator::operator() (const ManifestInstaller& installer1, const ManifestInstaller& installer2)
     {
         // Todo: Compare only architecture for now. Need more work and spec.
-        if (Utility::IsApplicableArchitecture(installer1.Arch) < Utility::IsApplicableArchitecture(installer2.Arch))
+        if (Utility::IsApplicableArchitecture(installer1.Arch) > Utility::IsApplicableArchitecture(installer2.Arch))
         {
-            return false;
+            return true;
         }
 
-        return true;
+        return false;

```

Comment:
>false [](start = 15, length = 5)

Why did you need to invert this?

Reply:
I want to make equal case return false.

---
In reply to: [467208009](https://github.com/microsoft/winget-cli/pull/527#discussion_r467208009) [](ancestors = 467208009)